scripts/plugin-tests.js

```diff
@@ -2441,10 +2442,18 @@ let syncTests = {
         },
       ],
     })
+    const outputs = Object.keys(metafile.outputs).map(outputPath => {
+      const absolutePath = path.resolve(outputPath)
+      const relativePath = path.relative(testDir, absolutePath)
```

Comment:
[sand] I believe `path.relative()` already applies to `path.resolve()` to both arguments, so the line above might not be needed.

Reply:
The path is originally relative to CWD, so first I'm converting it to an absolute path and then making it relative to the test path. Does that make sense?

Reply:
I believe if `outputPath` is a relative path, then `path.relative()` will assume it is relative to `cwd`, i.e. this additional step might not be needed. This appears to be the behavior when trying it locally.
The code for [`path.relative()`](https://github.com/nodejs/node/blob/6fe89a1fa6c10e1b68fe5c5d388a9d3e56bec138/lib/path.js#L503) seems to indicate this too.
This is not a problem though, especially considering this inside a test file.